### PR TITLE
[ODS-4920] Move data access functionality from EdFi.Ods.Sandbox to EdFi.Admin.DataAccess

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj [hunk 1]
```diff
@@ -24,4 +24,7 @@
   <ItemGroup>
     <None Include="../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Security\" />
+  </ItemGroup>
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj:29]: This should not be necessary

File: Application/EdFi.Admin.DataAccess/Repositories/AccessTokenClientRepo.cs [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                            .Value,
+                        out int duration))
+                    {
+                        duration = DefaultDuration;
+                    }
+
+                    return duration;
+                });
+
+
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/Repositories/AccessTokenClientRepo.cs:44]: remove extra white space please

File: Application/EdFi.Ods.Api/EdFi.Ods.Api.csproj [hunk 3]
```diff
@@ -42,8 +42,8 @@
     <EmbeddedResource Include="Services\Metadata\Resources\*.json" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\EdFi.Admin.DataAccess\EdFi.Admin.DataAccess.csproj" />
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api/EdFi.Ods.Api.csproj:45]: Should the ODS/API have a reference to Admin? I think this is not correct.

File: Application/EdFi.Ods.Tests/EdFi.Ods.Api/Controllers/ControllerHelper.cs [hunk 4]
```diff
@@ -5,24 +5,25 @@
...
     {
-        public static TokenController CreateTokenController(IClientAppRepo clientAppRepo,
-            IApiClientAuthenticator apiClientAuthenticator)
+        public static TokenController CreateTokenController(
+            IClientAppRepo clientAppRepo,
+            IApiClientAuthenticator apiClientAuthenticator,
+            IAccessTokenClientRepo accessTokenClientRepo)
         {
-            var tokenRequestProvider = new ClientCredentialsTokenRequestProvider(clientAppRepo, apiClientAuthenticator);
+            var tokenRequestProvider = new ClientCredentialsTokenRequestProvider(clientAppRepo, apiClientAuthenticator, accessTokenClientRepo);
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Tests/EdFi.Ods.Api/Controllers/ControllerHelper.cs:26]: What is driving this change?

**Review by jamessmckay — CHANGES_REQUESTED** [review]

File: Application/EdFi.Ods.Tests/EdFi.Ods.Api/Controllers/ControllerHelper.cs (hunk 4 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.Tests/EdFi.Ods.Api/Controllers/ControllerHelper.cs:26]: ClientCredentialsTokenRequestProvider requires IAccessTokenClientRepo accessTokenClientRepo

File: Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj (hunk 1 above)
Comment (semalaiappan) [reply Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj]: I removed it . Thanks

File: Application/EdFi.Admin.DataAccess/Repositories/AccessTokenClientRepo.cs (hunk 2 above)
Comment (semalaiappan) [reply Application/EdFi.Admin.DataAccess/Repositories/AccessTokenClientRepo.cs]: I removed it . Thanks

**Review by jamessmckay — APPROVED** [review]

**Pull request merged**